src/test/java/strategy/impl/BalanceOperationImplTest.java

```diff
@@ -0,0 +1,45 @@
+package strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import db.Storage;
+import model.FruitTransaction;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationImplTest {
+    private static final String BANANA = "banana";
+    private static final int OPERATION_AMOUNT = 10;
+    private static FruitTransaction fruitTransaction;
+
+    @BeforeClass
+    public static void beforeAll() {
+        Storage.map.clear();
+    }
+
+    @Test
+    public void handle_balance_ok() {
+        fruitTransaction = new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                BANANA, OPERATION_AMOUNT);
+        new BalanceOperationImpl().handler(fruitTransaction);

```

Comment:
Create this as an object on a class level and reuse it. Fix in all places, where you have the same solution